docker/Dockerfile

```diff
@@ -0,0 +1,6 @@
+FROM tomcat:8.0-jre8-alpine
+RUN rm -rf /usr/local/tomcat/webapps
+COPY webapps /usr/local/tomcat/webapps
+COPY docker-entrypoint.sh /usr/local/bin/docker-entrypoint.sh
```

Comment:
After the last copy, you'll probably need this...
```
RUN chmod +x /usr/local/bin/docker-entrypoint.sh
```

Reply:
I see in other Dockerfiles that they don't do that. I guess it's because it creates a new hash of the image. In order to ensure this file is executable I've run: `git update-index --chmod=+x docker-entrypoint.sh`